--- origsrc/libvirt-glib-0.2.3/python/Makefile.am	2013-10-07 05:26:44.000000000 -0500
+++ src/libvirt-glib-0.2.3/python/Makefile.am	2016-06-27 21:34:59.300818400 -0500
@@ -17,7 +17,7 @@ libvirtglibmod_la_CFLAGS = -Wno-redundan
 
 libvirtglibmod_la_LDFLAGS = -module -avoid-version \
 	$(CYGWIN_EXTRA_LDFLAGS)
-libvirtglibmod_la_LIBADD = ../libvirt-glib/libvirt-glib-1.0.la
+libvirtglibmod_la_LIBADD = ../libvirt-glib/libvirt-glib-1.0.la \
 	$(CYGWIN_EXTRA_LIBADD) $(CYGWIN_EXTRA_PYTHON_LIBADD)
 
 
